tests/manage/z_cluster/test_must_gather.py

```diff
@@ -137,11 +137,14 @@ def test_must_gather(self):
 
         # Verify that files for command output in json are present as expected
         commands_json = os.listdir(json_output_dir)
-        assert sorted(constants.MUST_GATHER_COMMANDS_JSON) == sorted(commands_json), (
+        assert sorted(constants.MUST_GATHER_COMMANDS_JSON) in sorted(commands_json), (
             f"Actual and expected json output commands files are not "
             f"matching.\nActual: {commands_json}\n"
             f"Expected: {constants.MUST_GATHER_COMMANDS_JSON}"
         )
+        if sorted(constants.MUST_GATHER_COMMANDS_JSON) != sorted(commands_json):
+            logger.warning("There are more actual must gather"
```

Comment:
shouldn't the test fail here?

Reply:
it will fail at assertion before if there is any command that are in expected but not in actual, warning will pop in case that all expected cmds exist, but there is more commands. i may change this logic due to my comment here: https://github.com/red-hat-storage/ocs-ci/issues/2560#issuecomment-677453286